### DISCUSSION TITLE
Is there any way to continue GitHub Copilot conversation on GitHub CLI?

### DISCUSSION THREAD

**Discussion by mthlpbs** [community/community#138664] [Copilot Conversations]
I'm exploring ways to continue an interactive GitHub Copilot conversation directly within the GitHub CLI. Has anyone found a method or workaround to maintain ongoing interactions with Copilot in a CLI environment?

**Comment by asurpbs** [comment]
Currently, GitHub Copilot's conversational features are designed for editor environments like VS Code. There's no direct way to interact with Copilot via the GitHub CLI, but you can generate suggestions in the editor and use them in the CLI as a workaround.

**Comment by smashcraft92** [comment]
You can't in the CLI.  But you can continue a conversation in the online web based Github Copilot.  I havent tested this with other models but I know it works with GPT-4.1. If you ask the A.I. if is possible to continue where you left off it will say no and then give you a bunch of reasons why not.  However, you can achieve this by utilizing the share button at the top of the conversation page on the right. 

![share button location](https://github.com/user-attachments/assets/bf24ce4e-ceef-4a60-b339-4bde929fa01c)

Once you click it you will git a share conversation window with an empy text box, press the green share button.

![Share Conversation link](https://github.com/user-attachments/assets/de28d533-577f-4b63-8aca-463b9b23de93)

Copy and save the link it generates.  Then even if you log out you can access that conversation again with the link.

![continue](https://github.com/user-attachments/assets/14f81fc4-82c1-4be7-9983-0a33931f378d)

please continue conversation utilizing text from before the " Messages beyond this point will start a new private conversation" note.  And it will continue as if you never left.

![confirmation](https://github.com/user-attachments/assets/02684663-1e3a-4ef6-b2dc-7cde9c1dde41)

Hope this helps.
